server/controllers/admin/users/index.js

```diff
@@ -0,0 +1,269 @@
+/**
+ * @overview Users
```

Comment:
True - depends on the fuzzy search algorithm I guess.  In theory, `users/index.js` should be similar to `users.js`, just with extra stuff in the middle.

I think the benefits of using `index.js` outweigh the negatives - but would be happy to have a discussion about it.